easypermissions/src/test/java/pub/devrel/easypermissions/AppSettingsDialogTest.java

```diff
@@ -187,15 +187,14 @@ public void shouldNegativeListener_whenClickingPositiveButtonFromSupportFragment
     private void setUpActivityAndFragment() {
         activityController = Robolectric.buildActivity(TestActivity.class)
                 .create().start().resume();
-        fragmentController = SupportFragmentController.of(new TestFragment())
-                .create().start().resume();
+        fragmentController = new FragmentController<>(TestFragment.class);
 
         spyActivity = Mockito.spy(activityController.get());
-        spyFragment = Mockito.spy(fragmentController.get());
+        spyFragment = Mockito.spy(fragmentController.resume());
```

Comment:
Can we do the same for the activityController? If so, we will be able to get rid of the tearDownActivityAndFragment method.